src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -3076,13 +3092,13 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
                     pResult.codePointerOrStubLookup.constLookup.accessType = InfoAccessType.IAT_PVALUE;
                     pResult.codePointerOrStubLookup.constLookup.addr = (void*)ObjectToHandle(
 #if READYTORUN
-                        _compilation.NodeFactory.InterfaceDispatchCell(targetMethod, new ModuleToken(_tokenContext, pResolvedToken.token)
+                        _compilation.NodeFactory.InterfaceDispatchCell(targetMethod, new ModuleToken(_tokenContext, pResolvedToken.token), isUnboxingStub: false
 #else
                         _compilation.NodeFactory.InterfaceDispatchCell(targetMethod
-#if !SUPPORT_JIT
+#endif // READYTORUN
+#if !SUPPORT_JIT || READYTORUN
```

Comment:
Why do we need the `|| READYTORUN` part?

Reply:
> Why do we need the || READYTORUN part in CorInfoImpl around line
3100? To let us pass around call site names to interface dispatch
cells - as interface dispatch cells are per call site even though
they might reference the same interface method, we need some way
to distinguish their symbols as we validate symbol uniqueness to
maintain output consistency.

Does R2R compiler define `SUPPORT_JIT`? `SUPPORT_JIT` should not be defined - it indicates CorInfoImpl behaviors needed for the JIT prototype (System.Private.Jit). Once R2R stops defining it, this `if` check will be true (because the condition is "not experimental JIT prototype").